tests/test_deploy_github_pages.py

```diff
@@ -411,8 +428,8 @@ def test_abort_source_branch_only_exists_locally(setup_test_env):
                                                  "--module_path", "../test_package", "../another_test_package"])
 
     assert e.match(f"Source branch exists locally, but not on remote, and source branch is not current branch."
-                   f"Please push your source branch to remote.")
-    assert e.type == SystemExit
+                   f"Please push your source branch to remote.") \
+           and e.type == SystemExit
```

Comment:
You may also wan't to consider making it a `multi line assert` using the `()` syntax.